crates/fuzzing/wasm-spec-interpreter/README.md

```diff
@@ -0,0 +1,44 @@
+wasm-spec-interpreter
+=====================
+
+This project shows how to use `ocaml-interop` to call into the Wasm spec interpreter. There are
+several steps to making this work:
+ - building the OCaml Wasm spec interpreter as a static library
+ - building a Rust-to-OCaml FFI bridge using `ocaml-interop` and a custom OCaml wrapper
+ - linking both things into a Rust crate
+
+### Dependencies
+
+This crate only builds in an environment with:
+- `make` (the Wasm spec interpreter uses a `Makefile`)
+- `ocamlopt`, `ocamlbuild` (available with, e.g., `dnf install ocaml`)
+- Linux tools like `cp` and `ar`; currently it is easiest to build the static libraries in a single
+  environment but this could be fixed in the future (TODO)
+
+Remember to retrieve the Wasm spec submodule:
+
+```
+git clone ... --recursive 
```

Comment:
`git submodule update --init` after you cloned works too.

Reply:
Yeah, there's multiple ways to do this and an experienced Git user will probably know of several of them. Are you OK resolving this comment or are you getting at something else here?